### PR TITLE
nu e varianta finala, trebuie pozele mai mici

### PR DESCRIPTION
**Pull request by ruxi983**

### CODE REVIEW

File: client/src/components/ActivitatiSecundare.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import bg_tft from "../assets/img/webp/activitatisecundare/teamfight.webp";
+
+export default function JocuriSecundare()
+{
+    return (
+    
+        <ul id="cards">
+            <li className="lol">
+                <a target="_blank" href="https://support.riotgames.com/hc/en-us">
+                    <div className="w-full h-full"><img src={bg_tft}/></div>
```
Comment (cristim67) [inline client/src/components/ActivitatiSecundare.js]: all images should also contain the alt attribute 

https://www.w3schools.com/tags/att_img_alt.asp

File: client/src/components/ActivitatiSecundare.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+                </a>
+
+            </li>
+            
+        </ul>    
+    
+    
+    
+        );
```
Comment (cristim67) [inline client/src/components/ActivitatiSecundare.js]: get rid of new lines and try to use prettier

https://www.educative.io/answers/how-to-set-up-prettier-and-automatic-formatting-on-vs-code

File: client/src/views/Home.js [hunk 3]
```diff
@@ -29,7 +30,9 @@ export default function Home() {
 
           <div id="Inscrieri" className=" pt-0">
           <ButoanePrincipale/>
-          {/* Act. secundare */}
+
+          <ActivitatiSecundare/>
+
```
Comment (cristim67) [inline client/src/views/Home.js]: you still have some newlines that don't make sense, you should delete them manually if your priettier doesn't delete them

File: client/src/components/ActivitatiSecundare.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+export default function JocuriSecundare()
+{
+    return (
+    
+        <ul id="cards">
+            <li className="lol">
+                <a target="_blank" href="https://support.riotgames.com/hc/en-us">
+                    <div className="w-full h-full"><img src={bg_tft} alt="Imagine TFT-TeamFight Tactics"/></div>
+                    <div className="hover">
+                        <img src={logo_tft}></img>
```
Comment (cristim67) [inline client/src/components/ActivitatiSecundare.js]: tag img need alt attribute

**Review by cristim67 — APPROVED** [review]
when you are satisfied you can merge.

**Comment by AMihneaa** [comment]
Branch vechi si inactiv

**Pull request closed without merging**